cmd/kube-apiserver/app/server.go

```diff
@@ -673,8 +673,17 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
 	}
 
 	if s.Etcd.EnableWatchCache {
+		sizes := serveroptions.DefaultWatchCacheSizes()
```

Comment:
This is basically recovering the behavior from: https://github.com/kubernetes/kubernetes/pull/91818/files#diff-170c324cc9e649c56eba833146296ea83b4fe433cb1c05b63ca18578609da4d2L666